src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -733,6 +749,28 @@ public function collectFromCode($content, $fileName, Module $module)
                     continue;
                 }
 
+                // Allow _t(Entity.Key, 'translation', $var) and expand _t(Entity.Key, $var)
+                if ($id == T_VARIABLE && !empty($currentEntity)) {
+                    // We have a translation, eg: _t(Entity.Key, 'translation', $var)
+                    if (count($currentEntity) == 2) {
+                        continue;
+                    }
```

Comment:
How does having exactly two items mean we have a translation? Are the two items the key and the default text?
And is the continue because it doesn't matter what's in any subsequent arguments to the `_t()` function?

Reply:
the current entity is an array with the following entries: name, translation, comments

if count = 1 : the translation key is being defined, but no translation has been found yet
if count = 2 : a translation has been stored for this key
if count = 3 : there is a comment

to be honest it's not very intuitive and it would deserve to be refactored to something less arbitrary (some kind of structered data) but that's how it works at the moment and i didn't want to change everything